modules/charset/breakwriter.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package charset
+
+import (
+	"bytes"
+	"io"
+)
+
+// BreakWriter wraps an io.Writer to always write '\n' as '<br>'
+type BreakWriter struct {
+	io.Writer
+}
+
+// Write writes the provided bs transparently replacing '\n' with '<br>'
```

Comment:
> Write writes the provided bs

Yes, on the first read I absolutely read that wrong.
Perhaps we should rename this variable to `bytes`?